docs/source/cpp/tutorials/basic_arrow.rst

```diff
@@ -141,7 +141,7 @@ Building int16 Arrays
 
 An :class:`ArrayBuilder` has its type specified at the time of declaration.
 Once this is done, it cannot have its type changed. We have to make a new one when we switch to year data, which
-requires a 16-bit integer at the minimum. Of course, there's an :class:`ArrayBuilder` for that.
+requires a 16-bit integer at the minimum. Of course, there’s an :class:`ArrayBuilder` for that.
```

Comment:
This is not a correct / expected change?

Reply:
Thanks for catching this. When rebasing, I grabbed the wrong change. The above change undid the change in https://github.com/apache/arrow/pull/40006 so I'll push up a fix.

Reply:
Reverted in 19e01b7e5529b1a8fb56f18b9281d23c95af6cba.